docs/modules/configuration/pages/tests-configuration.adoc

```diff
@@ -142,6 +142,10 @@ e.g., `0`, `#`, specify the way indices are displayed.
 |`false`
 |Skips the story in case of failure in the given story
 
+|`bdd.configuration.meta-by-row`
+|`false`
+|Enabled filtering of scenario by meta defined in examples table rows
```

Comment:
it's not really clear, actually it doesn't enable filtering, it allows to set different meta per ExamplesTable row

Reply:
No, you can set meta even without this prop but it won't be taken into consideration when filtering stories
<img width="787" alt="image" src="https://github.com/user-attachments/assets/96dad7f2-73a0-4ecd-8897-dbda91ebacf5">